src/api/flaskr/service/profile/funcs.py

```
@@ -625,46 +728,46 @@ def update_user_profile_with_lable(
+            .first()
         )
-        .order_by(UserProfile.id.desc())
-        .first()
-    )
-    if user_profile:
-        return user_profile.profile_value
-    else:
-        return None
+        if user_value:
+            return user_value.value
+    except Exception as exc:  # pragma: no cover - defensive fallback
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)